js/src/hooks/usePolling.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useApiFetchCallback from './useApiFetchCallback';
+import useCountdown from '.~/components/contact-information/phone-number-card/useCountdown';
+
+/**
+ * A hook for polling to an API Endpoint on intervals
+ *
+ * @param {import('@wordpress/api-fetch').APIFetchOptions} options Options to be forwarded to `apiFetch`.
+ * @param {number} [delay=20] Time in seconds to wait for each polling request
+ * @param {boolean} [stopOnResolved=false] If true. It stops the polling when the data is received
+ * @return {Object} Payload containing the result of the API call as well as a trigger function for starting the polling.
+ */
+export default function usePolling(
+	options,
+	delay = 5,
+	stopOnResolved = false
+) {
+	const { second, callCount, startCountdown } = useCountdown();
```

Comment:
After leaving the Attribute Mapping page, it will cause an error `Cannot read properties of undefined (reading 'id')` in `useCountdown` and lead to a blank page. Considering `useCountdown` was changed to have the default `handle` value in the `develop` branch, we could cherry-pick f3cc0437ae9dd124aaac702c3b14099490e121ec into this PR or merge the `develop` branch to the shared branch of this project and rebase related PRs. So that the use case here could keep no parameters.